charts/kubernetes-external-secrets/templates/rbac.yaml

```diff
@@ -14,6 +17,7 @@ rules:
     verbs: ["create", "update"]
   - apiGroups: [""]
     resources: ["namespaces"]
+    resourceNames: [{{ .Release.Namespace | quote }}]
```

Comment:
I think the app will still work after adding this line, because you added
```  if (scopeNamespace) {
    watchEndpoint = kubeClient.apis[customResourceManifest.spec.group]
      .v1.watch.namespaces(scopeNamespace)[customResourceManifest.spec.names.plural]
  }
```
If that's right, we could remove `"list"` from `verbs`. 
If I'm wrong and this is a breaking change, it'll show up in tests and I'll delete this line